pages/4_Documentation.py

```diff
@@ -323,6 +323,35 @@
          ( 'N.A.'           , 'The mutation site could not be predicted (for instance, because it is located in an unstructured region)') ]
 st.table(pd.DataFrame(data, columns=['Value', 'Meaning']))
 
+st.subheader("""Long range: AlloSigma2-PSN""")
+
+st.write("""This module is available only for ensemble mode and is designed to 
+validate the predictions of AlloSigma2 using a Protein Structure Network (PSN)-based method.
+
+The module relies on the output of the Long Range module, specifically the mutations predicted 
+to affect pocket residues. These predictions are then used as input for identifying potential 
+communication paths between mutations and their respective pocket sites using PyInteraph2. 
```

Comment:
here:
  - PyInteraph2 could be a link to the paper
  - you can briefly explain what the PyInteraph analysis does (and specify it's performed on the ensemble)